package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "objection-password",
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
Given my above comments about async/await and the spread syntax, I think we should bump this version to 3.0.0 and set the `engines` property to be `>= 8.3.0`. Thoughts?

Reply:
Good point

Reply:
Actually, I just took a look at `bcrypt` and version 5.0.0 and it has `engines` set to node `>=10.0.0` so that's probably a safer bet, right?

And the `bcrypt` README actually suggests `>=12.0.0` 🤔

Reply:
Good catch! Hmm yeah, not all of the versioning lines up on their side. Given that we're rocking bcrypt `5.x`, let's set the minimum to Node `12.x` just so we're not shooting ourselves in the foot unless you have objections.